src/sentry/runner/commands/cleanup.py

```diff
@@ -197,6 +197,7 @@ def is_filtered(model):
     # Deletions that use `BulkDeleteQuery` (and don't need to worry about child relations)
     # (model, datetime_field, order_by)
     BULK_QUERY_DELETES = [
+        (models.EventMapping, 'date_added', None),
```

Comment:
Add `-date_added` instead of `None`. That's the `order_by` argument which we need.